app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+from app.models import Actor

```

Comment:
follow import style according to PEP8
https://peps.python.org/pep-0008/#imports

Reply:
![6](https://user-images.githubusercontent.com/113552706/223503164-02765037-2c1d-42f0-a27b-475b491264da.PNG)
as i understood, you were refering to this part